modules/test/playwright/package.json

```diff
@@ -9,6 +10,7 @@
 		"@liferay/npm-scripts": "49.2.0",
 		"@playwright/test": "1.39.0",
 		"@types/node": "^20.9.0",
+		"ts-node": "^10.9.2",
```

Comment:
Nice! I wrote my scripts in JavaScript, but I may recosider to move them to typescript if we use this dependency.